ledger/participant-state/kvutils/tools/src/main/scala/ledger/participant/state/kvutils/tools/integritycheck/UpdateNormalizer.scala

```diff
@@ -72,7 +73,24 @@ object RejectionReasonDescriptionNormalizer extends UpdateNormalizer {
   }
 }
 
-/** Rejection reason strings should always be ignored as they can change arbitrarily. */
+/** Rejection reasons `Disputed`, `InvalidLedgerTime` and `Inconsistent` are being conflated
+  *  as their usage has been re-designed in SDK 1.16.0-snapshot.20210727.7476.0.b5e9d861 [KVL-1015].
+  */
+object RejectionReasonNormalizer extends UpdateNormalizer {
```

Comment:
Can we come up with a more specific name? Perhaps `RejectionsForConsistencyNormalizer`?

Reply:
What about `CommandRejectionRedesignNormalizer`?

Reply:
(I know it's a history-dependent name but updates normalizers are inherently historical byproducts)

Reply:
Sure, works for me.

Reply:
399012a